common/datasets/librispeech/export.py

```diff
@@ -126,7 +137,11 @@ def _export_legacy_bpe(output_prefix):
     )
 
 
-def export_all(output_prefix):
+def export_all(
+    output_prefix: str,
+    returnn_python_exe: Optional[tk.Path] = None,
```

Comment:
This does not make sense to be configurable, the `export_all` (and other export functions) are explicitly what the `asruser` repo should export. If you say we should have a "second" version of exporting .ogg zip files without the settings entry we can do that, but then this would be explicit here and not a configurable parameter.

Reply:
so we should always symlink from the common work dir for `export` functions?!

Reply:
Yes exactly, the `export` function define what can be symlinked.